doc/contributing/security-model-strategy.md

```diff
@@ -0,0 +1,83 @@
+# Security Model Strategy
+
+A clear security model and features like permissions and policy enforcement
+has been identified as one of the
+[top technical priorities](https://github.com/nodejs/node/blob/master/doc/contributing/technical-priorities.md#permissionspoliciessecurity-model)
+for the future success of Node.js.
```

Comment:
```suggestion
of Node.js.
```

Since this document is presumably a permanent thing in the repo, we should not talk about "future success" and things. Either it's a priority for us or it isn't. When the future arrives, it will not stop being a priority, presumably.